barretenberg/cpp/src/barretenberg/commitment_schemes/kzg/kzg.hpp

```diff
@@ -75,13 +77,13 @@ template <typename Curve> class KZG {
      *      - P₀ = C − v⋅[1]₁ + r⋅[W(x)]₁
      *      - P₁ = [W(x)]₁
      */
-    static std::array<GroupElement, 2> compute_pairing_points(const OpeningClaim<Curve>& claim,
-                                                              const auto& verifier_transcript)
+    static VerifierAccumulator reduce_verify(const OpeningClaim<Curve>& claim, const auto& verifier_transcript)
```

Comment:
my attempt at starting to unify interfaces for Zeromorph, IPA verifier will also call `reduce_verify` in a subsequent PR and return a VerifierAccumulator that for now is set to `bool` (we always perform the full IPA verification) until we have the IPA Accumulators implemented